client/my-sites/plugins/plugin-site/plugin-site.jsx

```diff
@@ -21,4 +24,24 @@ const PluginSite = ( props ) => {
 	return <PluginSiteJetpack { ...props } />;
 };
 
-export default PluginSite;
+export default connect(
+	( state, ownProps ) => {
+		const pluginSlug = get( ownProps, 'plugin.slug' );
+		const siteId = get( ownProps, 'site.ID' );
+
+		if ( includes( [ 'jetpack', 'vaultpress' ], pluginSlug ) && isSiteAutomatedTransfer( state, siteId ) ) {
+			return {
+				...ownProps,
+				...{
```

Comment:
Oh if these are new props, we can just return them directly from mapStateToProps. You'd just remove the default prop values in the component.